### PR TITLE
FAQ addition

### PR DESCRIPTION
**Pull request by Fretzi**

### CODE REVIEW

File: FAQ/en.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+We announce sign-ups before they open, but until then, you will have to wait. Sign-ups are being held during specific time periods, and are not open at all time.
+
+### Can I play without being part of the Quaver Alpha?
+
+Yes, you can download an offline version from Quaver’s [Github page](https://github.com/Quaver/Quaver/releases), However, you won’t be able to experience any of the online features.
+
+### Does this mean I can’t download maps from the Quaver website?
+
+That is correct, however, Adri; a member of Quaver offers an alternate download [here](https://rhythmgamers.net/pack/) in the form of Map Packs for anyone to access. Keep in mind he doesn’t update those packs very frequently.
```
Comment (YaLTeR) [inline FAQ/en.md]: This `;` does not belong here.
```suggestion
That is correct, however, Adri (a member of the Quaver team) offers an alternate download [here](https://rhythmgamers.net/pack/) in the form of map packs for anyone to access. Keep in mind he doesn’t update those packs very frequently.
```

File: FAQ/en.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+### Is there an estimate for the public release?
```
Comment (YaLTeR) [inline FAQ/en.md]:
```suggestion
### Is there an estimate for the public release date?
```

File: FAQ/en.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+We do not have an estimate for a public release yet.
```
Comment (YaLTeR) [inline FAQ/en.md]:
```suggestion
We do not have an estimate for the public release date yet.
```

File: FAQ/en.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## Gameplay
+
+### What key modes does Quaver support?
+
+Quaver currently only supports 4 keys and 7 keys.
+
+### Can I import maps from other games?
+
+Yes, Quaver currently supports .osu and .sm files and we are working on increasing the amount of files supported in the near future.
```
Comment (YaLTeR) [inline FAQ/en.md]:
```suggestion
Yes, Quaver currently supports .osu and .sm files and we are working on supporting more file formats.
```

File: FAQ/en.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+### I have a lot of maps in an external game, can I import them all at once?
+
+It is possible to import all of your maps at once by linking the games “.db” file inside your “quaver.cfg” file, which can be located inside your Quaver folder.
```
Comment (YaLTeR) [inline FAQ/en.md]:
```suggestion
It is possible to import all of your maps at once by linking the game’s “.db” file inside your “quaver.cfg” file, which can be located inside your Quaver folder.
```

File: FAQ/en.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+### Can I import my skin from another game?
+
+Yes, you can download a converter by Adri [here](https://rhythmgamers.net/QBC/) and a tutorial can be found [here](https://www.youtube.com/watch?v=pWeLbx48NVI).
+
+### I found a bug, where can I report it?
+
+Any bug reports should be posted on our [Github](https://github.com/Quaver/Quaver/issues). It doesn't matter how minor you think the issue is. Make sure to post anything you find, but make sure to check for already existing threads to avoid duplication.
```
Comment (YaLTeR) [inline FAQ/en.md]:
```suggestion
Any bug reports should be posted on our [GitHub issues](https://github.com/Quaver/Quaver/issues). It doesn't matter how minor you think the issue is. Make sure to post anything you find, but please check if the issue has already been reported to avoid duplication.
```

File: FAQ/en.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+### I have a feature request, where can I post it?
+
+Feature requests belong, just like bug reports, on our [Github](https://github.com/Quaver/Quaver/issues). Simply choose “Feature Request” when creating a new issue!
```
Comment (YaLTeR) [inline FAQ/en.md]:
```suggestion
Feature requests belong, just like bug reports, on our [GitHub issues](https://github.com/Quaver/Quaver/issues). Simply choose “Feature Request” when creating a new issue!
```

File: FAQ/en.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+
+## Troubleshooting
+
+### The game doesn’t launch
+
+Make sure Steam is running, as Steam is required in order to launch Quaver. This also applies to the offline build!
```
Comment (YaLTeR) [inline FAQ/en.md]:
```suggestion
Make sure Steam is running, as Steam is required in order to run Quaver. This also applies to the offline build!
```

File: FAQ/en.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+### The game is stuck on the welcome screen
+
+If the game doesn’t advance from the welcome screen, there might be a Steam-wide issue. Try again a bit later if this occurs.
```
Comment (YaLTeR) [inline FAQ/en.md]: This one is already fixed in the next update, so I'm not sure if this point should be kept.

Comment (Fretzi) [reply FAQ/en.md]: I guess we can remove it then.

File: FAQ/en.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
+# Frequently Asked Questions
+
+## General
+
```
Comment (Swan) [inline FAQ/en.md]: More questions to add:

1. What is Quaver?
2. How is Quaver different from other games?
3. What platforms will Quaver be released on?

File: FAQ/en.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+### What state is Quaver currently in?
```
Comment (Swan) [inline FAQ/en.md]: Rename this question to `How far in development is Quaver?`

File: FAQ/en.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+
+Quaver is currently in the closed Alpha state. In order to use online features of Quaver, you need to sign up for the Alpha.
```
Comment (Swan) [inline FAQ/en.md]: Reword this to: `Quaver is still in early development and is currently in closed alpha. `

File: FAQ/en.md [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+
+### Where can I sign up for the Quaver Alpha?
+
+We announce sign-ups before they open, but until then, you will have to wait. Sign-ups are being held during specific time periods, and are not open at all time.
```
Comment (Swan) [inline FAQ/en.md]: `Registrations for the closed alpha occur in rounds. Each round of sign-ups are announced before they open.`

File: FAQ/en.md [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+
+### Can I play without being part of the Quaver Alpha?
+
+Yes, you can download an offline version from Quaver’s [Github page](https://github.com/Quaver/Quaver/releases), However, you won’t be able to experience any of the online features.
```
Comment (Swan) [inline FAQ/en.md]: `...won’t be able to experience any of the online features including online rankings, map uploading, and other community-based features such as in-game chat.`

File: FAQ/en.md [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+
+### Does this mean I can’t download maps from the Quaver website?
+
+That is correct, however, Adri (a member of the Quaver team) offers an alternate download [here](https://rhythmgamers.net/pack/) in the form of map packs for anyone to access. Keep in mind he doesn’t update those packs very frequently.
```
Comment (Swan) [inline FAQ/en.md]: You cannot download maps from the Quaver website, however you can still download packs via a [mirror](https://rhythmgamers.net/pack/).

File: FAQ/en.md [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+
+### Is there an estimate for the public release date?
+
+We do not have an estimate for the public release date yet.
+
+
+## Gameplay
+
+### What key modes does Quaver support?
```
Comment (Swan) [inline FAQ/en.md]: `What game modes does Quaver support?`

File: FAQ/en.md [hunk 17]
```diff
@@ -0,0 +1,61 @@
...
+
+Quaver currently only supports 4 keys and 7 keys.
```
Comment (Swan) [inline FAQ/en.md]: Quaver currently only supports 4 keys (4k) and 7 keys (7k).

File: FAQ/en.md [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+
+### Can I import maps from other games?
+
+Yes, Quaver currently supports .osu and .sm files and we are working on supporting more file formats.
```
Comment (Swan) [inline FAQ/en.md]: `Yes, Quaver currently supports .osz and .sm files. To import them, drag the files into the game window.`.

File: FAQ/en.md [hunk 19]
```diff
@@ -0,0 +1,61 @@
...
+
+### I have a lot of maps in an external game, can I import them all at once?
+
+It is possible to import all of your maps at once by linking the game’s “.db” file inside your “quaver.cfg” file, which can be located inside your Quaver folder.
+
+### Can I import my skin from another game?
+
+Yes, you can download a converter by Adri [here](https://rhythmgamers.net/QBC/) and a tutorial can be found [here](https://www.youtube.com/watch?v=pWeLbx48NVI).
```
Comment (Swan) [inline FAQ/en.md]: The game doesn't support skin conversions out of the box, however you can download a .osk skin converter [here](https://rhythmgamers.net/QBC/) along with a [tutorial](https://www.youtube.com/watch?v=pWeLbx48NVI).

File: FAQ/en.md [hunk 20]
```diff
@@ -0,0 +1,61 @@
...
+
+### I found a bug, where can I report it?
+
+Any bug reports should be posted on our [GitHub issues](https://github.com/Quaver/Quaver/issues). It doesn't matter how minor you think the issue is. Make sure to post anything you find, but please check if the issue has already been reported to avoid duplication.
```
Comment (Swan) [inline FAQ/en.md]: `It doesn't matter how minor you think the issue is.` remove this

File: FAQ/en.md [hunk 21]
```diff
@@ -0,0 +1,61 @@
...
+
+## Troubleshooting
+
+### The game doesn’t launch
+
+Make sure Steam is running, as Steam is required in order to run Quaver. This also applies to the offline build!
+
+### I can’t upload my mapset using the editor
+
+If your mapset fails to upload using the client, you can upload it manually on the [Upload Mapset](/upload/mapset/) page.
```
Comment (Swan) [inline FAQ/en.md]: Link is broken

File: FAQ/en.md (hunk 10 above)
Comment (Fretzi) [reply FAQ/en.md]: Added all 3.

File: FAQ/en.md (hunk 11 above)
Comment (Fretzi) [reply FAQ/en.md]: Done

File: FAQ/en.md (hunk 12 above)
Comment (Fretzi) [reply FAQ/en.md]: Done

File: FAQ/en.md (hunk 13 above)
Comment (Fretzi) [reply FAQ/en.md]: Applied

File: FAQ/en.md (hunk 14 above)
Comment (Fretzi) [reply FAQ/en.md]: Applied

File: FAQ/en.md (hunk 15 above)
Comment (Fretzi) [reply FAQ/en.md]: Applied

File: FAQ/en.md (hunk 16 above)
Comment (Fretzi) [reply FAQ/en.md]: Applied

File: FAQ/en.md (hunk 17 above)
Comment (Fretzi) [reply FAQ/en.md]: Applied

File: FAQ/en.md (hunk 18 above)
Comment (Fretzi) [reply FAQ/en.md]: Applied

File: FAQ/en.md (hunk 19 above)
Comment (Fretzi) [reply FAQ/en.md]: Applied

File: FAQ/en.md (hunk 20 above)
Comment (Fretzi) [reply FAQ/en.md]: Done

File: FAQ/en.md (hunk 21 above)
Comment (Fretzi) [reply FAQ/en.md]: Changed the link to a full quavergame.com link

File: FAQ/en.md [hunk 22]
```diff
@@ -0,0 +1,73 @@
+# Frequently Asked Questions
+
+## General
+
+### What is Quaver?
+
+Quaver is a community-driven rhythm game, aiming to bring VSGRs to the next level. The game is open to players of all skill levels, casual and seasoned.
```
Comment (Swan) [inline FAQ/en.md]: Quaver is a community-driven and open source competitive rhythm game. It is inspired by a variety of classic rhythm games and plans to expand on them by being focused on real-time competitive play.

File: FAQ/en.md [hunk 23]
```diff
@@ -0,0 +1,73 @@
...
+
+### How is Quaver different from other games?
+
+The game is fully open-source, allowing anyone to help and improve the game. Furthermore, we are aiming to provide a new VSGR experience by bringing competitive concepts, such as competitive matchmaking.
```
Comment (Swan) [inline FAQ/en.md]: The game is fully open-source, which allows*

VSRG*

File: FAQ/en.md [hunk 24]
```diff
@@ -0,0 +1,73 @@
...
+
+### Does this mean I can’t download maps from the Quaver website?
+
+You cannot download maps from the Quaver website, however you can still download packs via a [mirror](https://rhythmgamers.net/pack/).
+
+### Is there an estimate for the public release date?
+
+We do not have an estimate for the public release date yet.
+
+
```
Comment (Swan) [inline FAQ/en.md]: Remove the extra line break here

File: FAQ/en.md [hunk 25]
```diff
@@ -0,0 +1,73 @@
...
+The game doesn't support skin conversions out of the box, however you can download a .osk skin converter [here](https://rhythmgamers.net/QBC/) along with a [tutorial](https://www.youtube.com/watch?v=pWeLbx48NVI).
+
+### I found a bug, where can I report it?
+
+Any bug reports should be posted on our [GitHub issues](https://github.com/Quaver/Quaver/issues). Make sure to post anything you find, but please check if the issue has already been reported to avoid duplication.
+
+### I have a feature request, where can I post it?
+
+Feature requests belong, just like bug reports, on our [GitHub issues](https://github.com/Quaver/Quaver/issues). Simply choose “Feature Request” when creating a new issue!
```
Comment (Swan) [inline FAQ/en.md]: Similar to bug reports, feature requests belong on our....

File: FAQ/en.md (hunk 22 above)
Comment (Fretzi) [reply FAQ/en.md]: applied

File: FAQ/en.md (hunk 23 above)
Comment (Fretzi) [reply FAQ/en.md]: applied and oops

File: FAQ/en.md (hunk 24 above)
Comment (Fretzi) [reply FAQ/en.md]: Also removed the line break above Troubleshooting.

File: FAQ/en.md (hunk 25 above)
Comment (Fretzi) [reply FAQ/en.md]: done
